docs/articles/expensify-classic/getting-started/Plan-Types.md

```diff
@@ -2,4 +2,31 @@
 title: Plan-Types
 description: Plan-Types
 ---
-## Resource Coming Soon!
+# Overview
+You can access comprehensive information about Expensify's plans and pricing by visiting www.expensify.com/pricing. Below, we provide an overview of each plan type to assist you in selecting the one that best suits your business or personal requirements.
```

Comment:
```suggestion
You can access comprehensive information about Expensify's plans and pricing by visiting [www.expensify.com/pricing](www.expensify.com/pricing). Below, we provide an overview of each plan type to assist you in selecting the one that best suits your business or personal requirements.
```
This will make it a clickable link